agent/consul/streaming_endpoint.go

```diff
@@ -117,21 +131,30 @@ func sendEvent(event stream.Event, eventFilter stream.EventFilterFunc, aclFilter
 	id := event.ID()
```

Comment:
IMO we should do the hashing here with a comment about why and the assumptions about collision probability and safety. etc.